src/TwigLoader.php

```diff
@@ -30,7 +30,10 @@ public function __construct($paths = [], ModuleHandlerInterface $module_handler,
     parent::__construct($paths, $module_handler, $theme_handler);
 
     $default_theme = $theme_handler->getTheme($theme_handler->getDefault());
-    if (!$default_theme->base_theme || !$base_theme = $theme_handler->getTheme($default_theme->base_theme)) {
+    if ($base_theme = $default_theme->base_theme) {
```

Comment:
Note that theoretically there can be multiple base themes; i.e., a base theme is a child of another base theme, so this would have to be a while loop until there are no more base themes.

However, as that facility is very rarely used, we can move forward with this as-is.

Reply:
Yes, I know. I had a loop before, but decided to go with the one base theme solution for now as I didn't know, if this is really necessary.

For completion, and future reference find the unused code:
```php
foreach (array_keys($default_theme->base_themes) as $base_theme) {
  $base_theme = $theme_handler->getTheme($base_theme);
  if (!isset($base_theme->info['component-libraries'])) {
    continue;
  }
  foreach ($base_theme->info['component-libraries'] as $namespace => $component) {
    $paths = isset($component['paths']) ? $component['paths'] : [];
    foreach ($paths as $path) {
      $this->prependPath($default_theme->getPath() . '/' . $path, $namespace);
    }
// […]